Code/Framework/AzToolsFramework/Tests/Prefab/Overrides/PrefabInspectorOverrideTestFixture.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/std/smart_ptr/unique_ptr.h>
+#include <AzToolsFramework/Prefab/Overrides/PrefabOverridePublicInterface.h>
+#include <AzToolsFramework/Prefab/PrefabFocusPublicInterface.h>
+#include <Prefab/PrefabTestFixture.h>
+#include <Prefab/Overrides/PrefabOverrideTestFixture.h>
+
+namespace AzToolsFramework
+{
+    class EntityPropertyEditor;
+}
+
+namespace UnitTest
+{
+    using namespace AzToolsFramework::Prefab;
```

Comment:
I generally avoid this in code but I think an exception can be granted for tests and test fixtures. This fixture particularly tests prefab overrides by enabling prefab feature flags and talking to the prefab systems apis. It has very less chances of running into namespace collisions due to dependencies on other systems. Using fully qualified names will be an overkill and make the code really verbose and annoying to write imo. If anything, I'd argue that we change the namespace of this class itself to be 'AzToolsFramework::Prefab' instead of 'UnitTest' if we want to stop doing 'using namespace' . I picked 'UnitTest' because that's what most test fixtures seem to be using but I think it's okay to not use it for every fixture?